src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx

```diff
@@ -5,37 +5,41 @@ tags:
   - New Relic Mobile Android
   - Install configure
 metaDescription: 'How to configure the New Relic Android agent to report app launch time'
-freshnessValidatedDate: never
+freshnessValidatedDate: 2024-06-11
 ---
 
-With the Android agent, you can report app launch time, also known as "cold time" and "hot time." 
-
-## Configure app launch time [#app-launch-time]
-
 <Callout variant="important">
 This feature is available for agent versions 6.9.0 and higher.
 </Callout>
 
-To enable the Android agent to report app launch time: Add the following to your application's `AndroidManifest.xml`:
-   ```xml
-   <application>
-
-     <!-- make sure to provide unique authorities if apply to multiple apps on the same device -->
-     <provider
-        android:name="com.newrelic.agent.android.rum.contentprovider.NewRelicAppContentProvider"
-        android:authorities="com.newrelic.android.agent.nrprovider.${applicationId}"
-        android:initOrder="200"
-        android:exported="false"/>
-        
-   </application>
-   ```
-   
-Your app launch time is reported with the `AppLaunch/Cold` and `AppLaunch/Hot` [metrics](/docs/data-apis/understand-data/new-relic-data-types/#metrics). To track these launch time metrics, you can use the <DoNotTranslate>**Query builder**</DoNotTranslate> to create a custom dashboard. To use the <DoNotTranslate>**Query builder**</DoNotTranslate>, go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Capabilities > Query builder**</DoNotTranslate>.
-
-Here's a sample query for capturing cold and hot launch times: 
-
-```sql
-  SELECT average(newrelic.timeslice.value ) AS 'AppLaunch/Cold' FROM Metric WHERE metricTimesliceName = 'AppLaunch/Cold' AND entity.guid = 'YOUR_APP_TOKEN' SINCE 1664218800000 UNTIL 1664220600000 TIMESERIES
+With the Android agent, you can report app launch time, also known as "cold time" and "hot time." 
+
+## Configure app launch time [#configure-app-launch-time]
+
+To enable the Android agent to report app launch time, addd the following to your application's `AndroidManifest.xml`:
+
+```xml
+<application>
+
+    <!-- make sure to provide unique authorities if apply to multiple apps on the same device -->
```

Comment:
For this line "if apply to multiple apps on the same device" should it be "if it applies?" or "if applying?"